src/playground-project.ts

```diff
@@ -801,12 +801,17 @@ const expandProjectConfig = async (
       filePromises.push(
         (async () => {
           const resp = await fetch(new URL(filename, baseUrl).href);
+          let contentType = resp.headers.get('Content-Type')?.toLowerCase() ?? 'text/plain';
+          // native fetch api will default to octet-stream for unknown filetypes
```

Comment:
I think it'll be the server doing this, not fetch.

The file extension fallback logic already exists elsewhere, so no need to set the mime type here. But treating octet-stream the same as undefined seems like it makes sense.

Reply:
feel the same way, `application/mime-type` is historically a bit bucket

Reply:
removed the injection